### DISCUSSION TITLE
Treat Everything Button for Clinic

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#8969] [Ideas and suggestions]
Honestly I feel like it should've existed without any questions asked. The treatments are all too cheap that you can generally heal all your crew for under 300 marks in pretty bad case scenario.

**Comment by AshEdwardsDesign** [comment]
+1

I'd also like to see this implemented. 

It can get tedious in SP with a large AI crew where everyone has some injuries to click each crew member and add them to the treatment list.
